.travis.yml

```
@@ -35,12 +35,12 @@ jobs:
         - BUILD_NUMBER=$TRAVIS_BUILD_NUMBER
         - TESTAPP_TAG=master
       cache: false
+      language: minimal
       before_install: skip
-      install:
-        - "pip install awscli"
+      install: skip
       before_script:
-        - "aws s3 cp s3://optimizely-travisci-artifacts/ci/trigger_fullstack-sdk-compat.sh ci/ && chmod u+x ci/trigger_fullstack-sdk-compat.sh"
+        - mkdir $HOME/travisci-tools && pushd $HOME/travisci-tools && git init && git pull https://$CI_USER_TOKEN@github.com/optimizely/travisci-tools.git && popd
       script:
-        - "ci/trigger_fullstack-sdk-compat.sh"
+        - "$HOME/travisci-tools/fsc-trigger/trigger_fullstack-sdk-compat.sh"
       # travis_terminate not supported by darwin
       after_success: skip
```

```suggestion
      after_success: travis_terminate 0
```
we can terminate now that we arent using darwin